src/runWarmUp.cpp

```diff
@@ -35,7 +35,8 @@ void runWarmUp(DateVector timestring, int nYears){
 	double out_res;
 	double RiverInflow;
 	double RoutedOutflowCell;
-	
+	NumericVector K_release(array_size);
+	K_release.fill(1);
```

Comment:
Please use fill with 0.1 instead of 1, as it is the default release value for low storage. Or you use the definition used above, but then I would prefer creating a function out of it and using the function as it is then 3 times present in the code.